src/components/lotes/LoteForm.jsx

```diff
@@ -77,26 +79,66 @@ class LoteForm extends Component {
   }
 
   updateLote() {
+    const { id, firebase } = this.props;
     const { title, price, artista, obras, tecnica } = this.state;
 
     this.setState({ isLoading: true })
     this.props.setIsLoading();
 
-    this.props.firebase
-      .update(`lotes/${this.props.id}`, {
+    firebase
+    // Update Lote
+      .update(`lotes/${id}`, {
         title,
         price,
         artista,
         obras,
         tecnica,
       })
+    // Fetch all Catalogos
+      .then(() => {
+        return firebase.ref(`catalogos`).once('value');
+      })
+    // Find the Catalogo that contains the Lote we just updated
+      .then((dataSnapshot) => {
+        const catalogos = dataSnapshot.val();
```

Comment:
could do `const catalogos = dataSnapshot.val() || {};` so you don't get an undefined object